contributing.md

```diff
@@ -0,0 +1,29 @@
+## Contributing
```

Comment:
Ok. By the way, I'm not sure how new this feature is and if you're aware of it, but GitHub has "Allow edits from maintainers". I worked a company with a custom version control solution before that allowed reviewers to make commits/changes to pull requests and it was super efficient (especially for typo fixes or non-conflicting changes). Loved it, works as long as everyone's cool with having their work modified (like in this case).